doc/api/util.md

```diff
@@ -10,6 +10,68 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
+`callbackify` takes an `async function` (or a function that returns a Promise)
```

Comment:
Nittest of nits: MDN uses "`async` function" backticks only around `async`, which is IMO more consistent since we don't use `function`.